locales/en-US/app.l20n

```diff
@@ -16,27 +14,27 @@
 
 <menuWiki "Test Pilot Wiki">
 <menuFileIssue "File an Issue">
-<menuLogout "Logout">
-<menuRetire "Retire">
+<menuLogout "Sign out">
+<menuRetire "Leave Test Pilot">
 
 <landingFxaAlternateButton "Sign in">
-<landingIntroOne "Test out new features.">
+<landingIntroLead "Go beyond...">
+<landingIntroOne "Test new features.">
 <landingIntroTwo "Give us feedback.">
 <landingIntroThree "Help build Firefox.">
 <landingFxaGetStartedButton "Get started with a Firefox Account">
 <landingLegalNotice "By proceeding, you agree to the <a>Terms of Service</a> and <a>Privacy Notice</a> of Test Pilot.">
-<landingPageFooterCopy "More tests coming soon!">
-<landingInstallHeaderOne "Install the Test Pilot Add-on">
-<landingInstallHeaderTwo "and you're good to go!">
-<landingInstallButton "Install the Add-on">
+
+<landingInstallHeader "Get the add-on to get going!">
+<landingInstallButton "Install Test Pilot">
 <landingInstallingButton "Installing Test Pilot...">
 <landingInstallMessage "Test Pilot installed!">
-<landingInstalledButton "Let's Go">
+<landingInstalledButton "Choose your features">
 
-<landingDownloadFirefoxDesc "(of course, you'll need Firefox to use Test Pilot)">
+<landingDownloadFirefoxDesc "(Test Pilot is available for Firefox on Windows, OSX and Linux)">
```

Comment:
Not immediately relevant, but there are also rumors that this may get rebranded to "MacOS" eventually (as early as June): http://techcrunch.com/2016/04/17/the-next-version-of-os-x-could-be-called-macos/